src/pages/research.astro

```diff
@@ -0,0 +1,1170 @@
+    publicationInfo: {
+      type: "conference",
+      conference:
+        "11th International Workshop on Software Configuration Management (SCM-11)",
+      location: "Portland, Oregon, USA",
+      pages: "102–117",
+    },
+    preprintOrArchiveUrl: "https://nixos.org/~eelco/pubs/iscsd-scm11-final.pdf",
+  },
+];
+
+const sponsors = await getEntry("sponsors", "info");
+
+const papersByYear = papers.reduce((acc, paper) => {
+  validatePaper(paper);
+  const year = paper.year;
+  if (!acc[year]) acc[year] = [];
+  acc[year].push(paper);
+  return acc;
+}, {});
+
+const sortedYears = Object.keys(papersByYear).sort((a, b) => b - a);
+
+function getVenueString(pubInfo: PublicationType) {
+  switch (pubInfo.type) {
+    case "conference":
+      return `${pubInfo.conference}, ${pubInfo.location}`;
+    case "journal":
+      return `${pubInfo.journal}${pubInfo.volume ? `, vol. ${pubInfo.volume}` : ""}${pubInfo.number ? `, no. ${pubInfo.number}` : ""}`;
+    case "thesis":
+      return `${pubInfo.thesisType} thesis, ${pubInfo.institution}, ${pubInfo.location}`;
+  }
+}
+---
+
+<Layout title="Research and Scientific Publications">
+  <PageHeader text="Research and Scientific Publications" />
+  <Container class="grid px-12 pt-16">
+    <p class="text-xl font-bold text-gray-500">
+      Nix is a research project who's time to solve today's software
+      distribution issues has come.
+    </p>
+  </Container>
+
+  <Container class="grid md:grid-cols-2 gap-8 px-12 pt-8 pb-20">
+    <div
+      class="flex flex-col justify-center font-extralight gap-4 leading-relaxed"
+    >
+      <p>
+        Nix got started as a research project by Eelco Dolstra and his
+        collaborators, which over time attracted a dedicated community based on
+        its revolutionary ideas.
+      </p>
+      <p>
+        As a part of this growing community, a new crop of researchers has
+        discovered Nix as a fitting tool to tackle todays issues of scientific
+        and binary reproducibility, software distribution and supply chain
+        security.
```

Comment:
```suggestion
        Nix started as a research project by Eelco Dolstra and his collaborators at Utrecht University around 2003.
        Since then, scientists from multiple institutions have published their work on repeatable computation and reliable, secure software distribution.
      </p>
      <p>
        This collection traces the continued exploration of theoretical foundations and practical applications of the ideas underlying Nix and its ecosystem.
```

I'd want to go away from using flashy adjectives, and find a phrasing that doesn't go out of date quickly.